NetTopologySuite.Windows.Forms.Test/BasicTestMedia.cs

```diff
@@ -1,374 +1,374 @@
-            if (!isValidOp.IsValid)
-                Assert.IsTrue(false, isValidOp.ValidationError.ToString());
-
-            Assert.AreEqual(wpfGeom.GetArea(), ntsGeom.Area, 1e-5);
-        }
-
-        static BasicTestMediaPath()
-        {
-            var gf = Geometries.GeometryFactory.Default;
-            _multiPoint = gf.CreateMultiPoint(
-                new[]
-                    {
-                        new Coordinate(100*_random.NextDouble(), 100*_random.NextDouble()),
-                        new Coordinate(100*_random.NextDouble(), 100*_random.NextDouble()),
-                        new Coordinate(100*_random.NextDouble(), 100*_random.NextDouble()),
-                        new Coordinate(100*_random.NextDouble(), 100*_random.NextDouble()),
-                        new Coordinate(100*_random.NextDouble(), 100*_random.NextDouble())
-                    });
-        }
-
-        private static readonly Random _random = new Random(55942);
-        private static readonly IMultiPoint _multiPoint;
-
-        private static void PerformTest(IPointToPathGeometryFactory factory)
-        {
-            var gpw = new WpfPathGeometryWriter(new IdentityPointTransformation(), factory);
-            var res = gpw.ToShape(_multiPoint);
-
-            WpfGeometryToImage(res, string.Format("WPF-Path-MultiPoint-{0}.png", factory.GetType().Name));
-        }
-
-        private static readonly Size BitmapSize = new Size(300, 300);
-
-        private static void WpfGeometryToImage(WpfGeometry geom, string fileName)
-        {
-            var tmp = geom;
-
-            var pen = new Pen(Brushes.Brown, 2);
-            var bounds = tmp.GetRenderBounds(pen);
-            var scaleX = (BitmapSize.Width - 4)/bounds.Width;
-            var scaleY = (BitmapSize.Height - 4)/bounds.Height;
-
-            var scale = Math.Min(scaleX, scaleY);
-            var tmpMat = new MatrixTransform(scale, 0, 0, scale, -bounds.Left*scale, -bounds.Top*scale);
-
-            //tmp.Transform = tmpMat;
-            
-            GeometryDrawing gd = new GeometryDrawing(Brushes.Red, new Pen(Brushes.Brown, 2d/scale), geom);
-            DrawingVisual dv = new DrawingVisual();
-            //dv.Transform = tmpMat;
-            using (var dc = dv.RenderOpen())
-            {
-                //dc.PushTransform(MatrixTransform);
-                dc.DrawRectangle(Brushes.White, null, new Rect(new Point(0, 0), BitmapSize));
-                dv.Transform = tmpMat;
-                dc.DrawDrawing(gd);
-            }
-            RenderTargetBitmap rtb = new RenderTargetBitmap((int)BitmapSize.Width, (int)BitmapSize.Height, 96, 96, PixelFormats.Pbgra32);
-            
-            rtb.Render(dv);
-
-            PngBitmapEncoder png = new PngBitmapEncoder();
-            png.Frames.Add(BitmapFrame.Create(rtb));
-            using (Stream stm = File.Create(fileName))
-            {
-                png.Save(stm);
-            }
-        }
-    }
-
+﻿using System;
```

Comment:
Something weird happened that marked this whole file as changed on GitHub... why?  Was it an encoding thing?  Line endings?  Not looking at it until it's resubmitted without every line being marked as deleted + added, or minimally an explanation why.